web-ui/src/main/resources/catalog/templates/admin/classification/categories.html

```diff
@@ -74,7 +74,7 @@
             <button
               type="button"
               class="btn btn-primary pull-right"
-              data-ng-disabled="!gnCategoryFrom.$dirty"
+              data-ng-disabled="!gnCategoryFrom.$valid && !gnCategoryFrom.$dirty"
```

Comment:
This doesn't seem to work, exceeding the 255 chars, the button to save the category is enabled. In the sources dialog, the code should be different as in that form works.